README.md

```diff
@@ -16,6 +16,8 @@ $ npm install @hypothesis/frontend-shared --save
 
 ### SASS
 
+Your project must have `sass` and `tailwind` dependencies installed.
```

Comment:
The Tailwind npm package is called `tailwindcss`.

Reply:
Thanks for the pointer! Updated.